Error when require()'ing through Browserify

Hey,

I'm `require()`ing `d3plus` on my main JS file, and getting this error when running `browserify`:

```
Error: Cannot find module './array/comparator.coffee' from '/Users/manuel/Work/opus-frontend/v1.0/node_modules/d3plus'
  at /usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:55:21
  at load (/usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
  at onex (/usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
  at /usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
  at Object.oncomplete (fs.js:107:15)
```

--- a/index.js
+++ b/index.js
@@ -1,0 +1,5 @@
+'use strict';
+
+require('./src/libs.coffee')
+
+module.exports = require('./src/init.coffee')

--- a/package.json
+++ b/package.json
@@ -2,11 +2,9 @@
   "name": "d3plus",
   "version": "1.9.1",
   "description": "Data visualization made easy. A javascript library that extends the popular D3.js to enable fast and beautiful visualizations.",
-  "main": "d3plus.full.js",
   "devDependencies": {
     "browserify": "^13.0.0",
     "coffee-script": "^1.9.3",
-    "coffeeify": "^2.0.1",
     "event-stream": "^3.3.1",
     "gulp": "^3.9.0",
     "gulp-chmod": "^1.2.0",
@@ -66,7 +64,8 @@
     "numeric": "^1.2.6",
     "simplify-js": "^1.2.1",
     "static-kdtree": "^1.0.1",
-    "topojson": "^1.6.24"
+    "topojson": "^1.6.24",
+    "coffeeify": "^2.0.1"
   },
   "filename": "d3plus.min.js",
   "npmName": "d3plus",

--- a/src/init.coffee
+++ b/src/init.coffee
@@ -2,8 +2,6 @@
 # @class d3plus
 ###
 d3plus         = {}
-window.d3plus  = d3plus if typeof window isnt "undefined"
-module.exports = d3plus
 
 ###*
 # The current version of **D3plus** you are using. Returns a string in
@@ -196,3 +194,6 @@ stylesheet = require "./client/css.coffee"
 message    = require "./core/console/print.coffee"
 if stylesheet "d3plus.css"
   message.warning "d3plus.css has been deprecated, you do not need to load this file.", d3plus.repo+"releases/tag/v1.4.0"
+
+window.d3plus  = d3plus if typeof window isnt "undefined"
+module.exports = d3plus

